stock_barcodes/wizard/stock_barcodes_read_picking.py

```diff
@@ -0,0 +1,343 @@
+# Copyright 2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+from odoo import api, _, fields, models
+from odoo.tools.float_utils import float_compare
+from odoo.exceptions import ValidationError
+from odoo.fields import first
+from odoo.addons import decimal_precision as dp
+
+_logger = logging.getLogger(__name__)
+
+
+class WizStockBarcodesReadPicking(models.TransientModel):
+    _name = 'wiz.stock.barcodes.read.picking'
+    _inherit = 'wiz.stock.barcodes.read'
+    _description = 'Wizard to read barcode on picking'
+
+    picking_id = fields.Many2one(
+        comodel_name='stock.picking',
+        string='Picking',
```

Comment:
If the same name between with the name of the field and the string name. 
Odoo will call it from the name of the field. You do not need to define the string name.